tests/unit/swagger/definitionsGeneration/definitions.spec.ts

```diff
@@ -131,7 +131,7 @@ describe('Definition generation', () => {
           'TestSubModelContainerNamespace.InnerNamespace.TestSubModelContainer2',
           'TestSubModel2',
           'TestSubModelNamespace.TestSubModelNS',
-          'TsoaTest.TestModel73',
+          'tsoaTest.TsoaTest.TestModel73',
```

Comment:
For safe name conflict handling, every type needs to use its fully namespaced name. Not accepted to think, that TsoaTest type is equal to another interface in the tsoaTest namespace